src/server/input_service.rs

```diff
@@ -648,11 +648,12 @@ fn map_keyboard_mode(evt: &KeyEvent) {
     let (click_capslock, click_numlock) = sync_status(evt);
 
     #[cfg(windows)]
-    {
+    let click_numlock = {
```

Comment:
It looks good, it would be even better if it could be written in the sync_status function